src/mac/component.rs

```diff
@@ -10,7 +10,8 @@ use sys::ffi;
 use ComponentExt;
 
 pub(crate) const COMPONENTS_TEMPERATURE_IDS: &[(&str, &[i8])] = &[
-    ("CPU", &['T' as i8, 'C' as i8, '0' as i8, 'P' as i8]), // CPU "TC0P"
+    ("CPU Die", &['T' as i8, 'C' as i8, '0' as i8, 'F' as i8]), // CPU Die "TC0F"
```

Comment:
The naming is really not great considering that this is "on" the CPU contrary to "CPU proximity". Maybe add a comment explaining that this is a "CPU Die", and giving a small description (I found explanations on what is [here](https://discussions.apple.com/thread/6666066)) and keep the text to "CPU"? But not sure if better either...

Reply:
What do you think about the new comments? I tried to keep it short and clean.
But "CPU" as the text sounds better to me, too.